frontend-site/src/views/Services.vue

```diff
@@ -9,7 +9,9 @@
               <v-flex xs5>
                 <div style="border: 1px solid white; height: 100%;">
                   <!-- TODO: turn into image/svg tag -->
-                  {{ service.image || 'No image specified' }}
+
+                <v-img :src="service.image"></v-img>
+                <!-- <p v-else>No Image Provided</p> -->
```

Comment:
Can remove the TODO and the commented `<p>` element as we're doing the TODO in this PR